src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.util.LottoGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Money {
+    private static final int LOTTO_PRICE = 1000;
+
+    private int money;
+
+    public Money(String money) {
+        validateNumber(money);
+        this.money = Integer.parseInt(money);
+    }
+
+    public List<Lotto> buyLotto(){
```

Comment:
`Money`에서 필요한 메서드일까요 🤔 
극단적인 예시지만... `Money`로 다른 것들도 살 수 있다면 `Money` 클래스가 굉장히 커질 것 같아요!

```suggestion
    public List<Lotto> buyLotto() {
```
Convention!

Reply:
`Lottos`에서 `buyLotto()`를 처리하는게 좋을 것 같긴한데 그럼 테스트를 위하여 생성자를 두개 생성해야 할 것 같습니다. 검증을 위하여 여러개의 생성자를 생성하는것도 좋은 방법일까요?